### PR TITLE
cmd/snap/debug/boot-vars: add opts for setting dir and/or uc20 vars

### PR DESCRIPTION
**Pull request by anonymouse64**
This will be helpful as we start porting more systems and need to write spread tests or for folks like me who just need to get information from a partially working system as we port things to it.

### CODE REVIEW

File: cmd/snap/cmd_debug_bootvars.go [hunk 1]
```diff
@@ -23,20 +23,27 @@ import (
...
 		"(internal) obtain the snapd boot variables",
 		"(internal) obtain the snapd boot variables",
 		func() flags.Commander {
 			return &cmdBootvars{}
-		}, nil, nil)
+		}, map[string]string{
+			"uc20": i18n.G("Whether to use uc20 boot vars or not"),
+			"dir":  i18n.G("What directory to look for boot variables in"),
+		}, nil)
 	cmd.hidden = true
```
Comment (bboozzoo) [inline cmd/snap/cmd_debug_bootvars.go:44]: Opened https://github.com/snapcore/snapd/pull/8491 to not hide the command on core.

File: cmd/snap/cmd_debug_bootvars.go [hunk 2]
```diff
@@ -23,20 +23,27 @@ import (
 	"github.com/jessevdk/go-flags"
 
 	"github.com/snapcore/snapd/boot"
+	"github.com/snapcore/snapd/i18n"
 )
 
-type cmdBootvars struct{}
+type cmdBootvars struct {
+	UC20 bool   `long:"uc20"`
+	Dir  string `long:"dir"`
```
Comment (bboozzoo) [inline cmd/snap/cmd_debug_bootvars.go:31]: This should probably be named `RootDir` and `rootdir`

File: cmd/snap/cmd_debug_bootvars.go [hunk 3]
```diff
@@ -23,20 +23,27 @@ import (
...
 func init() {
 	cmd := addDebugCommand("boot-vars",
 		"(internal) obtain the snapd boot variables",
 		"(internal) obtain the snapd boot variables",
 		func() flags.Commander {
 			return &cmdBootvars{}
-		}, nil, nil)
+		}, map[string]string{
+			"uc20": i18n.G("Whether to use uc20 boot vars or not"),
+			"dir":  i18n.G("What directory to look for boot variables in"),
```
Comment (bboozzoo) [inline cmd/snap/cmd_debug_bootvars.go:42]: and `What root directory to look..`

File: cmd/snap/cmd_debug_bootvars.go (hunk 2 above)
Comment (bboozzoo) [reply cmd/snap/cmd_debug_bootvars.go]: Pushed that.

**Comment by anonymouse64** [comment]
thanks for the patches @bboozzoo

**Pull request merged**